index.html

```diff
@@ -3,7 +3,129 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script
+      src="https://kit.fontawesome.com/39da8831d4.js"
+      crossorigin="anonymous"
+    ></script>
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
+    </style>
+
     <title>Document</title>
   </head>
-  <body></body>
+  <body class="fo">
+    <!-- Sidenav -->
+    <nav
+      id="sidenav-3"
+      class="left-0 relative top-0 z-[1035] h-screen w-60 -translate-x-full overflow-hidden bg-zinc-800 shadow-[0_4px_12px_0_rgba(0,0,0,0.07),_0_2px_4px_rgba(0,0,0,0.05)] data-[te-sidenav-hidden='false']:translate-x-0"
+      data-te-sidenav-init
+      data-te-sidenav-hidden="false"
+      data-te-sidenav-color="white"
+    >
+      <ul
+        class="relative m-0 list-none px-[0.2rem] font-semibold"
+        data-te-sidenav-menu-ref
+      >
+        <li class="relative p-6 pr-16 pl-5">
```

Comment:
@Bishakha1589 you have padding, padding-left and padding right on the same element here resulting in the logo not being centered, please change this by using only p-something, not pl and pr. the logo should always be centered unless stated otherwise explicitly